packages/dapp/components/lib/hooks/contracts/use-protocol-contracts.ts

```diff
@@ -148,6 +159,18 @@ const useProtocolContracts = () => {
   protocolContracts.twapOracleDollar3poolFacet = new ethers.Contract(diamondAddress, TWAPOracleDollar3poolFacetArtifact.abi, <Provider>provider);
   protocolContracts.ubiquityPoolFacet = new ethers.Contract(diamondAddress, UbiquityPoolFacetArtifact.abi, <Provider>provider);
 
+  // other related contracts
+  // const sushiSwapPool = await protocolContracts.managerFacet.sushiSwapPoolAddress();
+  // const sushiSwapPoolContract = new ethers.Contract(sushiSwapPool, SushiSwapPoolArtifact.abi, <Provider>provider);
+  // const UniswapV2PairContract = new ethers.Contract(await sushiSwapPoolContract.pair(), UniswapV2PairABI, <Provider>provider);
+  const UniswapV2PairContract = new ethers.Contract("0x41e087485f47538752A1195D984109cB8Dc0E429", UniswapV2PairABI, <Provider>provider);
```

Comment:
Shouldn't these addresses be passed in as variables?

Reply:
> Shouldn't these addresses be passed in as variables?

Let me pass them as variables